src/Compilers/CSharp/Test/Semantic/Semantics/ExpressionBodiedMemberTests.cs

```diff
@@ -747,25 +805,145 @@ public class C
             Assert.Equal("System.Int32 C.P1 { get; set; }", model.GetSymbolInfo(node).Symbol.ToTestDisplayString());
         }
 
+        [Fact]
```

Comment:
> [Fact] [](start = 7, length = 7)

Looking back at the change with the feature code (PR 13543), some additional test suggestions:
-features disabled (errors for IDS_FeatureExpressionBodiedAccessor and IDS_FeatureExpressionBodiedDeOrConstructor)
-a direct parsing test for expression and constructor/deconstructor